spec/features/admin_role_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
```

Comment:
Please delete these tests, since you've moved them into the model test (and this isn't a feature)